Clarify that fornax is free (#182)
nasa-fornax/fornax-documentation: documentation/quick_start.md

Reviewer: ```suggestion
To create a free Fornax Science Console and Community Forum account, complete the form at:
```

Reviewer: Note: each user starts out with 1000 free credits per year.  The dashboard shows the status of those credits based on individual usage.

:alt: JupyterLab opening page with some Fornax customizations, as deployed in the Fornax Science Console.
```

> At this point, the server you have chosen will begin accruing cost (charged to your credit allotment) even though you have not started any computations.

Reviewer: ```suggestion
At this point, the selected server will begin charging against your credit allotment, even though no computations have started yet.
```

just remove the word "cost"
